DirectProgramming/C++SYCL_FPGA/Tutorials/Features/optimization_levels/minimum_latency/README.md

```diff
@@ -0,0 +1,290 @@
+# Minimum Latency Flow
+
+This FPGA tutorial demonstrates how to compile your design with the minimum latency flow to achieve low latency at the cost of increased f<sub>MAX</sub>.
+
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | Linux* Ubuntu* 18.04/20.04 <br> RHEL*/CentOS* 8 <br> SUSE* 15 <br> Windows* 10
+| Hardware                          | Intel&reg; Programmable Acceleration Card (PAC) with Intel Arria&reg; 10 GX FPGA <br> Intel&reg; FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix&reg; 10 SX) <br> Intel&reg; FPGA 3rd party / custom platforms with oneAPI support <br> **Note**: Intel&reg; FPGA PAC hardware is only compatible with Ubuntu 18.04*
+| Software                          | Intel® oneAPI DPC++/C++ Compiler
+| What you will learn               | How to use the minimum latency flow to compile low-latency designs<br>How to manually override underlying controls set by the minimum latency flow
+| Time to complete                  | 20 minutes
+
+> **Note**: Even though the Intel DPC++/C++ OneAPI compiler is enough to compile for emulation, generating reports and generating RTL, there are extra software requirements for the simulation flow and FPGA compiles.
+>
+> For using the simulator flow, one of the following simulators must be installed and accessible through your PATH:
+> - Questa*-Intel® FPGA Edition
+> - Questa*-Intel® FPGA Starter Edition
+> - ModelSim® SE
+>
+> When using the hardware compile flow, Intel® Quartus® Prime Pro Edition must be installed and accessible through your PATH.
+
+## Purpose
+
+This FPGA tutorial demonstrates how to use the minimum latency flow to compile low-latency designs and how to manually override underlying controls set by the minimum latency flow. By default, the minimum latency flow tries to achieve lower latency at the cost of decresed f<sub>MAX</sub>, so this flow is a good starting point for optimizing latency-sensitive designs.
+
+To compile your design with the minimum latency flow, pass the `-Xsoptimize=latency` flag to the `icpx` command.
+
+The minimum latency flow implies the following compiler controls:
+- Disable hyper-optimized handshaking on Intel Stratix&reg; 10 and Intel Agilex&trade; devices
```

Comment:
Should it be on all devices?

Reply:
Hyper-optimized handshaking only exists on S10 and Agilex.

Reply:
Can we then mention: Disable hyper-optimized handshaking
Rather than specifying which devices support them? (see other comment about IPA)